tests/ZfrOAuth2Test/Server/Grant/AuthorizationGrantTest.php

```diff
@@ -244,7 +255,7 @@ public function testCanCreateTokenResponse($hasRefreshGrant)
 
         $response = $this->grant->createTokenResponse($request, new Client(), $owner);
 
-        $body = json_decode($response->getContent(), true);
+        $body = json_decode($response->getBody(), true);
```

Comment:
@weierophinney : I'm not sure to understand how things work.. This always return empty string. I've tried to use $response->getBody()->getContents(), but still empty string. When I construct the response, I do something like that: $response = new Response(json_encode(['foo' => 'bar']));

Is this the write way?